Tests/NIOSSLTests/SSLCertificateTest.swift

```diff
@@ -380,4 +380,20 @@ class SSLCertificateTest: XCTestCase {
 
         XCTAssertEqual(certBytes, expectedCertBytes)
     }
+    
+    func testPrintingDebugDetailsNoAlternativeNames() throws {
+        let expectedDebugDescription = "<NIOSSLCertificate;serial_number=9fd7d05a34ca7984;common_name=robots.sanfransokyo.edu>"
+        let cert = try assertNoThrowWithValue(NIOSSLCertificate(bytes: .init(samplePemCert.utf8), format: .pem))
+        let debugString = String(describing: cert)
+
+        XCTAssertEqual(debugString, expectedDebugDescription)
+    }
+    
+    func testPrintingDebugDetailsWithAlternativeNames() throws {
+        let expectedDebugDescription = "<NIOSSLCertificateserial_number=46231a526848d57af4999e29f89988d178d94da2;common_name=localhost;alternative_names=localhost,example.com,192.168.0.1,2001:db8::1>"
```

Comment:
`NIOSSLCertificateserial_number` 🤔

Reply:
Oh oops, I hadn't pushed a commit that fixed this haha